src/Microsoft.Identity.Client/Mats/Internal/Events/ApiEvent.cs

```diff
@@ -27,25 +27,39 @@ public enum ApiIds
         {
             None = 0,
 
-            AcquireTokenSilentWithAuthority = 31,
-            AcquireTokenSilentWithoutAuthority = 30,
-
-            AcquireTokenWithScope = 170,
-            AcquireTokenWithScopeHint = 171,
-            AcquireTokenWithScopeHintBehavior = 172,
-            AcquireTokenWithScopeHintBehaviorAuthority = 173,
-            AcquireTokenWithScopeUser = 176,
-            AcquireTokenWithScopeUserBehavior = 174,
-            AcquireTokenWithScopeUserBehaviorAuthority = 175,
-
-            AcquireTokenOnBehalfOfWithScopeUser = 520,
-            AcquireTokenOnBehalfOfWithScopeUserAuthority = 521,
-
-            AcquireTokenForClientWithScope = 726,
-            AcquireTokenForClientWithScopeRefresh = 727,
-            AcquireTokenByRefreshToken = 728,
-
-            AcquireTokenByAuthorizationCodeWithCodeScope = 830
+            //AcquireTokenSilentWithAuthority = 31,
+            //AcquireTokenSilentWithoutAuthority = 30,
+
+            //AcquireTokenWithScope = 170,
+            //AcquireTokenWithScopeHint = 171,
+            //AcquireTokenWithScopeHintBehavior = 172,
+            //AcquireTokenWithScopeHintBehaviorAuthority = 173,
+            //AcquireTokenWithScopeUser = 176,
+            //AcquireTokenWithScopeUserBehavior = 174,
+            //AcquireTokenWithScopeUserBehaviorAuthority = 175,
+
+            //AcquireTokenOnBehalfOfWithScopeUser = 520,
+            //AcquireTokenOnBehalfOfWithScopeUserAuthority = 521,
+
+            //AcquireTokenForClientWithScope = 726,
+            //AcquireTokenForClientWithScopeRefresh = 727,
+            //AcquireTokenByRefreshToken = 728,
+
+            //AcquireTokenByAuthorizationCodeWithCodeScope = 830,
+
+            // TODO: These are all new ids, one for each of the flows.
+            // If these arrive, then the permutuations of "with behavior/hint/scope/refresh/etc" are all
+            // bits sent as separate fields via ApiTelemetryFeature values.
+            AcquireTokenByAuthorizationCodeV2 = 1000,
```

Comment:
>V2 [](start = 43, length = 2)

do these need to have V2 on the end?

Reply:
Some of them overlap with existing ID values.  I'm basically defining new keys due to the addition of the feature bits.  Open to naming suggestions to not clash with the old values...?

Reply:
i see...just not a fan of having something specific in the name, like "v2"...but i guess it's okay. also can't think of anything better to suggest anyway

---
In reply to: [287117891](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/pull/1161#discussion_r287117891) [](ancestors = 287117891)

Reply:
I fixed this.  these are just our internal constants, so i removed the old ones (the commented out block), and remove v2 from the end of the names.